receipt/src/test/java/com/hyperwallet/android/ui/receipt/viewmodel/ListPrepaidCardReceiptViewModelTest.java

```diff
@@ -0,0 +1,50 @@
+package com.hyperwallet.android.ui.receipt.viewmodel;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.CoreMatchers.notNullValue;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+import com.hyperwallet.android.ui.receipt.repository.PrepaidCardReceiptRepository;
+import com.hyperwallet.android.ui.receipt.repository.PrepaidCardReceiptRepositoryImpl;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.robolectric.RobolectricTestRunner;
+
+@RunWith(RobolectricTestRunner.class)
+public class ListPrepaidCardReceiptViewModelTest {
+
+    private ReceiptViewModel mReceiptViewModelToTest;
+
+    @Before
+    public void initializedViewModel() {
+        PrepaidCardReceiptRepository prepaidCardReceiptRepository = new PrepaidCardReceiptRepositoryImpl(
+                "trm-ppc-token");
+        ListPrepaidCardReceiptViewModel.ListPrepaidCardReceiptViewModelFactory factory =
+                new ListPrepaidCardReceiptViewModel.ListPrepaidCardReceiptViewModelFactory(
+                        prepaidCardReceiptRepository);
+
+        mReceiptViewModelToTest = factory.create(ReceiptViewModel.class);
+    }
+
+    @Test
+    public void testIsLoadingData_liveDataNotNull() {
```

Comment:
Could we use the suffix "returnsLiveData" for all methods or "liveDataIsNotNull" ? I'd prefer the first because we're already using that prefix on other places (returnsXXX)